src/SprykerEco/Zed/Heidelpay/HeidelpayConfig.php

```diff
@@ -13,6 +13,8 @@
 
 class HeidelpayConfig extends AbstractBundleConfig implements HeidelpayConfigInterface
 {
+    public const MERCHANT_TRANSACTION_CONFIG_NOT_FOUND = '';
```

Comment:
Can we name this in a more business way?
Right now it's not clear why it's named like an exception.

Reply:
Also, can we make it `protected`?